Description: Switch Ubuntu from powerpc to powerpc-smp kernel
Author: Adam Conrad <adconrad@ubuntu.com>
Forwarded: no

--- live-build-3.0~a24.orig/scripts/build/lb_binary_debian-installer
+++ live-build-3.0~a24/scripts/build/lb_binary_debian-installer
@@ -352,7 +352,7 @@ then
 			DI_REQ_PACKAGES="yaboot"
 			case "${LB_MODE}" in
 				ubuntu)
-					DI_PACKAGES="${DI_REQ_PACKAGES} linux-image-powerpc linux-image-powerpc64-smp"
+					DI_PACKAGES="${DI_REQ_PACKAGES} linux-image-powerpc-smp linux-image-powerpc64-smp"
 					;;
 
 				*)
--- live-build-3.0~a24.orig/functions/defaults.sh
+++ live-build-3.0~a24/functions/defaults.sh
@@ -683,11 +683,11 @@ Set_defaults ()
 				ubuntu)
 					case "${LIST}" in
 						stripped|minimal)
-							LB_LINUX_FLAVOURS="${LB_LINUX_FLAVOURS:-powerpc}"
+							LB_LINUX_FLAVOURS="${LB_LINUX_FLAVOURS:-powerpc-smp}"
 							;;
 
 						*)
-							LB_LINUX_FLAVOURS="${LB_LINUX_FLAVOURS:-powerpc powerpc64-smp}"
+							LB_LINUX_FLAVOURS="${LB_LINUX_FLAVOURS:-powerpc-smp powerpc64-smp}"
 							;;
 					esac
 					;;
